clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -938,6 +947,7 @@ public void commitSync(final Map<TopicPartition, OffsetAndMetadata> offsets) {
         acquire();
         try {
             coordinator.commitOffsetsSync(offsets);
+            interceptors.onCommit(offsets);
```

Comment:
On second thought, maybe that's backwards. We might need to move the interceptor hook into coordinator.commitOffsetsSync(offsets). Otherwise, the interceptor won't see the automatic synchronous commits on rebalance and on close.